packages/vlossom/src/stores/layout-store.ts

```diff
@@ -0,0 +1,59 @@
+import { ref, type Ref } from 'vue';
+import type { BarLayout, DrawerLayout, DrawerLayouts } from '@/declaration';
+import { objectUtil } from '@/utils';
+
+export class LayoutStore {
+    private _header: Ref<BarLayout> = ref({ position: 'relative', height: '' });
+    private _footer: Ref<BarLayout> = ref({ position: 'relative', height: '' });
+    private _drawers: Ref<DrawerLayouts> = ref({
+        left: { isOpen: false, responsive: false, placement: 'left', size: '' },
+        top: { isOpen: false, responsive: false, placement: 'top', size: '' },
+        right: { isOpen: false, responsive: false, placement: 'right', size: '' },
+        bottom: { isOpen: false, responsive: false, placement: 'bottom', size: '' },
+    });
+
+    public static getDefaultLayout(): LayoutStore {
+        return new LayoutStore();
```

Comment:
P5: provide 하는 이유가 별도로 없다면 provide가 필요 없을 것 같아 보였어요

```suggestion
       // psudo singleton
       if(this._layoutStore) { return this._layoutStore; }
       return this._layoutStore = new LayoutStore();
```

Reply:
아, 하위에 inject 받는 쪽에서 fallback으로 필요해서 일단 만들어뒀어용
다른 새끼 layout 작업 진행할 때 그 부분까지 고려해서 리뷰 주십셩 ㅎㅎㅎ